lib/rspec/junklet/junk.rb

```diff
@@ -117,6 +118,8 @@ def junk(*args)
             generator = -> { rand(max-min) + min }
           when :bool
             generator = -> { [true, false].sample }
+          when :url
+            generator = -> { "http://#{::SecureRandom.uuid}.com" }
```

Comment:
Maybe recurse into `junk` here instead? I'm thinking about putting together a PR that doesn't use SecureRandom.